Sources/SourceControl/GitRepository.swift

```diff
@@ -203,14 +204,31 @@ public class GitRepository: Repository, WorkingCheckout {
     /// The path of the repository on disk.
     public let path: AbsolutePath
 
+    /// The (serial) queue to execute git cli on.
+    private let queue = DispatchQueue(label: "com.apple.swiftpm.gitqueue")
```

Comment:
Oh I see that Daniel already commented in the general comments (I missed it since it wasn't attached to this line)